bookkeeper-server/src/test/java/org/apache/bookkeeper/client/BookieRecoveryUseIOThreadTest.java

```diff
@@ -35,6 +37,7 @@ public BookieRecoveryUseIOThreadTest() {
         super(1);
     }
 
+    @BeforeEach
```

Comment:
I think we don't need this annotation, it's already on the base `BookKeeperClusterTestCase`